bitnami/redis/templates/configmap-scripts.yaml

```diff
@@ -280,6 +280,33 @@ data:
     {{ .Values.sentinel.preExecCmds | nindent 4 }}
     {{- end }}
     exec redis-server /opt/bitnami/redis-sentinel/etc/sentinel.conf --sentinel {{- if .Values.tls.enabled }} "${ARGS[@]}" {{- end }}
+  prestop-sentinel.sh: |
+    #!/bin/bash
+
+    . /opt/bitnami/scripts/libvalidations.sh
+
+    REDIS_SERVICE="{{ template "redis.fullname" . }}.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}"
```

Comment:
```suggestion
    REDIS_SERVICE="{{ include "redis.fullname" . }}.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}"
```

Reply:
The other scripts use "template". Is this a change for another PR or would you like to have the change here? So it is not in sync with the other scripts.

Reply:
Well, it is prefered to use `include` than `template` in this cases, it is not a big deal, but something we are changing little by little